packages/network/lib/cors.ts

```diff
@@ -124,67 +126,73 @@ function urlMatchesPolicyProps ({ policy, url, props }: {
   }
 }
 
-function urlMatchesPolicy ({ policy, url1, url2 }: {
+function urlMatchesPolicy ({ policy, frameUrl, topUrl }: {
   policy: Policy
-  url1: string
-  url2: string
+  frameUrl: string
+  topUrl: string
 }): boolean {
-  if (!policy || !url1 || !url2) {
+  if (!policy || !frameUrl || !topUrl) {
     return false
   }
 
   return urlMatchesPolicyProps({
     policy,
-    url: url1,
-    props: parseUrlIntoHostProtocolDomainTldPort(url2),
+    frameUrl,
+    topProps: parseUrlIntoHostProtocolDomainTldPort(topUrl),
   })
 }
 
-export function urlMatchesOriginProps (url, props) {
```

Comment:
I removed a couple of functions that were not used anywhere:
* urlMatchesOriginProps
* urlMatchesPolicyProps
* urlMatchesSameSiteProps

urlsSuperDomainOriginMatch and urlMatchesSuperDomainOriginProps are replaced by urlMatchesPolicyBasedOnDomain and urlMatchesPolicyBasedOnDomainProps